src/hosting/normalizedHostingConfigs.ts

```diff
@@ -1,31 +1,38 @@
-const _ = require("lodash");
+import * as _ from "lodash";
 
-function _filterOnly(configs, onlyString) {
+type HostingConfig = { site: string; target: string };
```

Comment:
Any reason to use a `type` here but use an `interface` for `TemplateServerResponse`?

Reply:
it's no reason. 
it seems more better to interface.

Reply:
please prefer interfaces to types. Maybe that's something I could enforce with the linter in the future...